gen/DocumentFormat.OpenXml.Generator.Models/Generators/Elements/DataModelWriterExtensions.cs

```diff
@@ -11,6 +11,132 @@ namespace DocumentFormat.OpenXml.Generator.Generators.Elements;
+    private static readonly Dictionary<string, Dictionary<string, List<string>>> _attributeData =
+        new Dictionary<string, Dictionary<string, List<string>>>()
+        {
+            // Example with annotations:
+            // {
+            //  This is the containing complex type class, in the json metadata, this comes from the fully qualified "Name": "c:CT_BubbleSer/c15:ser",
+            //  "c:CT_BubbleSer/c15:ser",
+            //  new Dictionary<string, List<string>>()
+            //  {
+            //        {
+            //              This is an example of obsoleting the whole class.
+            //              In the json metadata:
+            //                    Use the same fully qualified name as the class, for example "Name": "c:CT_BubbleSer/c15:ser",
+            //              "c:CT_BubbleSer/c15:ser",
+            //              new List<string>()
+            //              {
+            //              "[Obsolete(\"Unused property, will be removed in the next major version.\", false)]",
+            //              "[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)] ",
+            //              }
+            //        },
+            //        {
+            //              This is an example obsoleting a child element (property in C#)
+            //              In the json metadata:
+            //                    For child elements, this comes from "Name": "c:CT_PictureOptions/c:pictureOptions",
+            //              "c:CT_PictureOptions/c:pictureOptions",
+            //              new List<string>()
+            //              {
+            //              "[Obsolete(\"Unused property, will be removed in the next major version.\", false)]",
+            //              "[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)] ",
+            //              }
+            //        },
+            //        {
+            //              This is an example obsoleting a child attribute (property in C#)
+            //              In the json metadata: use for example "QName": ":formatCode",
+            //              ":formatCode",
+            //              new List<string>()
+            //              {
+            //              "[Obsolete(\"Unused property, will be removed in the next major version.\", false)]",
+            //              "[System.ComponentModel.EditorBrowsable(System.ComponentModel.EditorBrowsableState.Never)] ",
+            //              }
+            //        },
+            //  }
+            // },
+            {
+              "c:CT_BubbleSer/c15:ser",
+              new Dictionary<string, List<string>>()
+              {
+                    {
+                          "c:CT_PictureOptions/c:pictureOptions",
+                          new List<string>()
```

Comment:
These are duplicated, so it may be nice to have a readonly static list that can be reused